src/connection/neo4j/Neo4jConnectionModule.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Record as Neo4jRecord } from 'neo4j-driver';
+import { ConnectionModule } from '../ConnectionModule';
+import { runCypherQuery } from './runCypherQuery';
+import { extractQueryCallbacks, extractQueryParams } from './utils';
+
+const notImplementedError = (functionName: string): never => {
```

Comment:
To be resynced with the changes in https://github.com/neo4j-labs/neodash/pull/767

Reply:
ok, will look to sync with 767